### ISSUE TITLE
Update PR description instead of making a comment

### ISSUE THREAD

**Issue by rehanvdm** [corymhall/cdk-diff-action#37]
Will it be possible to post to the PR description instead of commenting on the PR? 

This will reduce noise, because GitHub notifies for every comment created on a PR.  I am not sure if the PR description will be the limiting factor here. If it does, maybe post to the PR description first, but if it is too long, truncate and create the rest as comments.

**Comment by corymhall** [comment]
Yeah I think this should be possible. I'll probably add a property you can configure for where you want the diff to go.

**Comment by rehanvdm** [comment]
Does a property like this exist yet?
